src/pages/admin/donate/components/donate-page-content/DonatePageContent.tsx

```diff
@@ -0,0 +1,291 @@
+import { useCallback, useEffect, useState } from 'react';
+import { useAdminClient } from '../../../../../hooks/admin/use-admin-client/useAdminClient';
+import { SupportOptionsApi } from '../../../../../services/api/admin/donate/support-options/support-options-api';
+import { CorrespondentBankDetailsApi } from '../../../../../services/api/admin/donate/correspondent-banks/correspondent-banks-api';
+import { SupportOptionsType } from '../../../../../types/admin/donate';
+import { DONATE_TEXT } from '../../../../../const/admin/donate';
+import { CategoryBar } from '../../../../../components/admin/category-bar/CategoryBar';
+import { GenericDetails } from '../generic-details/GenericDetails';
+import { SupportOptionsForm } from '../support-options/support-options-form/SupportOptionsForm';
+import {
+    Currencies,
+    useBankDetails,
+    mapCurrencyToBankCurrency,
+} from '../bank-details-currencies/currencies-manager/CurrenciesManager';
```

Comment:
As I mentioned before, `Currencies` & `mapCurrencyToBankCurrency` remove and use `BankCurrency` directly
```suggestion
import {
    useBankDetails,
} from '../bank-details-currencies/currencies-manager/CurrenciesManager';
```